recipes/hdbscan/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "0.8" %}
+
+package:
+  name: hdbscan
+  version: {{ version }}
+
+source:
+  fn: hdbscan-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/h/hdbscan/hdbscan-{{ version }}.tar.gz
+  sha256: ea09e8ad60a1734e46000e71cc4b4ca274629e5ed9c54015cf4d4150c1238a17
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - scikit-learn >=0.16
+    - cython >=0.17
+
+  run:
+    - python
+    - scikit-learn >=0.16
+
+test:
+  # Python imports
+  imports:
+    - hdbscan
+
+  # commands:
+    # You can put test commands to be run here.  Use this to test that the
+    # entry points work.
+
+
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  requires:
+    - nose
+    # Put any additional test requirements here.  For example
+    # - nose
+
+about:
+  home: http://github.com/lmcinnes/hdbscan
+  license: BSD
```

Comment:
It looks like the project is `BSD 3-Clause`, right? Can you confirm that and update it here?